LICENSE

```diff
@@ -1,4 +1,8 @@
-Copyright 2022 The KerasCV Authors. All rights reserved.
+Copyright © 2023 The KerasCV Authors
+All code in this repository excluding the code located in
+keras_cv/layers/preprocessing_3d/waymo is licensed under the Apache License,
+Version 2.0. The code appearing in the keras_cv/layers/preprocessing_3d/waymo
+folder is licensed under terms appearing below.
```

Comment:
This is just the Apache license. Should we direct them to `keras_cv/layers/preprocessing_3d/waymo/LICENSE`?